client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/item-info.js

```diff
@@ -51,10 +57,11 @@ ItemInfo.propTypes = {
 	openItemMove: PropTypes.func.isRequired,
 };
 
-const mapDispatchToProps = ( dispatch ) => {
-	return bindActionCreators( {
+export default connect(
+	( state ) => ( {
+		site: getSelectedSite( state ),
```

Comment:
Nitpick, but since this component gets passed already `siteId`, we should use that prop to get the `site` object.